avocado/utils/linux_modules.py

```diff
@@ -92,11 +99,31 @@ def loaded_module_info(module_name):
 
     :param module_name: Name of module to search for
     :type module_name: str
-    :return: Dictionary of module info, name, size, submodules if present
+    :return: Dictionary of module name, size, submodules if present, filename,
+             version, number of modules using it, list of modules it is
+             dependent on, list of params
```

Comment:
Of course, this particular docstring change is fine along with the change in the function implementation.